### PR TITLE
[feature #39] Add Save As/Open concordance

### PR DESCRIPTION
**Pull request by phmz**
Add feature #39

### CODE REVIEW

File: unitex/src/fr/umlv/unitex/frames/ConcordanceFrame.java [hunk 1]
```diff
@@ -61,10 +54,11 @@
 
 /**
  * This class describes a frame that can show an HTML concordance file.
- * 
+ *
  * @author Sébastien Paumier
  */
 public class ConcordanceFrame extends TabbableInternalFrame {
+  private final File f;
```
Comment (vahbuna) [inline unitex/src/fr/umlv/unitex/frames/ConcordanceFrame.java]: can we have a better name for the variable f? concordanceFile or concordanceFrameFile ?

**Comment by phmz** [comment]
Following your suggestion, I renamed the variable f to concordanceFile.

File: unitex/src/fr/umlv/unitex/frames/UnitexFrame.java [hunk 2]
```diff
@@ -170,6 +130,7 @@ public void textFrameOpened(boolean taggedText) {
 						convertFst.setEnabled(true);
 						exportTfstAsCsv.setEnabled(true);
 						closeText.setEnabled(true);
+            openConcordance.setEnabled(true);
```
Comment (martinec) [inline unitex/src/fr/umlv/unitex/frames/UnitexFrame.java]: Do not mix tabs with spaces for indentation. Use the same indentation character thought the code.

tab_spaces

File: unitex/src/fr/umlv/unitex/config/Config.java [hunk 3]
```diff
@@ -1256,24 +1272,24 @@ private static void chooseInitialLanguage() {
...
         final JComboBox langList = new JComboBox(langArr);
         if(defaultLangIndex != -1)
         	langList.setSelectedIndex(defaultLangIndex);
-        
+
         p.add(new JLabel("User: " + getUserName()));
         p.add(new JLabel("Choose the language you want"));
         p.add(new JLabel("to work on:"));
         p.add(langList);
-
```
Comment (martinec) [inline unitex/src/fr/umlv/unitex/config/Config.java]: Avoid code changes that neither fixes a bug or adds a feature...

File: unitex/src/fr/umlv/unitex/config/Config.java [hunk 4]
```diff
@@ -836,13 +852,13 @@ private static void setApplicationDir(String appPath, File s) {
...
     /**
      * Setup the UnitexToolLogger executable
-     * 
+     *
      * @param path
      *            external programs directory
      *
      * @author martinec
      */
-    public static File setupUnitexToolLogger(File path) {
```
Comment (martinec) [inline unitex/src/fr/umlv/unitex/config/Config.java]: Do not strip trailing spaces on the existing code as part of this PR. See http://blog.ranjansakalley.com/2015/09/cleaning-up-whitespace-changes-in-a-pull-request.html, for an approach on how-to cleaning up whitespace changes in a pull request. If you need to commit refactoring changes open a different PR prefixed with "[[refactor]](https://github.com/UnitexGramLab/unitex-doc-contributor-guidelines/blob/master/pages/05.guidelines/01.commits/docs.md)".

**Comment by martinec** [comment]
LGTM

**Pull request merged**